lib/test/unit/notification.rb

```diff
@@ -65,15 +65,17 @@ def included(base)
       # Notify some information.
       #
       # Example:
-      #   def test_notification
-      #     notify("I'm here!")
-      #     # Reached here
-      #     notify("Special!") if special_case?
-      #     # Reached here too
-      #   end
+      #
+      #     def test_notification
+      #       notify("I'm here!")
+      #       # Reached here
+      #       notify("Special!") if special_case?
+      #       # Reached here too
+      #     end
       #
       # options:
-      #   :backtrace override backtrace.
+      #
+      #     :backtrace override backtrace.
```

Comment:
This is not pre-formatted text. But we can work on this as follow-up pull request.